samples/dashboards/demo/minimal/demo.js

```diff
@@ -222,6 +262,18 @@ Dashboards.board('container', {
                 animation: false,
                 type: 'column',
                 spacing: [30, 30, 30, 20]
+            },
+            tooltip: {
+                valueSuffix: ' mcg',
+                stickOnContact: true
+            },
+            accessibility: {
+                chartContainerLabel: 'Vitamin A',
```

Comment:
- It looks like the Vitamin A and Iron is switched for the demos
- chartContainerLabel belongs in `lang`